src/main/java/no/nav/familie/inntektsmelding/integrasjoner/joark/JoarkTjeneste.java

```diff
@@ -76,8 +79,14 @@ private OpprettJournalpostRequest opprettRequest(String xmlAvInntektsmelding, In
             .medEksternReferanseId(UUID.randomUUID().toString())
             .medJournalfoerendeEnhet(JOURNALFØRENDE_ENHET)
             .medKanal(KANAL)
-            .medDokumenter(lagDokumenter(xmlAvInntektsmelding, pdf))
-            .build();
+            .medDokumenter(lagDokumenter(xmlAvInntektsmelding, pdf));
+
+        // Dette er for overstyring, siden fpsak ikke sender saksnummer ennå.
```

Comment:
Det vil også gjelde for arbeidsgiverinitiert inntektsmelding (da har vi ikke forespørsel) så trengs nok uansett :)

Reply:
Ja, dette er sant ;)